test/integration/api/v1/owner_test.rb

```diff
@@ -63,13 +63,13 @@ class Api::V1::OwnerTest < ActionDispatch::IntegrationTest
       params: { email: @other_user.email },
       headers: { "HTTP_AUTHORIZATION" => @other_user_api_key }
 
-    assert_response :unauthorized
+    assert_response :forbidden
 
     delete api_v1_rubygem_owners_path(@rubygem.slug),
       params: { email: @other_user.email },
       headers: { "HTTP_AUTHORIZATION" => @other_user_api_key }
 
-    assert_response :unauthorized
+    assert_response :forbidden
```

Comment:
I'm not totally certain about the effect of this slight change. It's because the old code made a special message when the api key owner didn't have ownership of the gem it was trying to change. This just merges into the general `forbidden` response now like most other actions.